query-algebrizer/src/clauses/where_fn.rs

```diff
@@ -308,17 +318,17 @@ mod testing {
         let good = vec![vec![TypedValue::Long(10), TypedValue::Boolean(true)],
                         vec![TypedValue::Long(11), TypedValue::Boolean(false)]];
         assert_eq!(reduced_rows_and_column_types(good.clone(), &vec![true, true]).expect("to reduce rows"),
-                   (good.clone(),
+                   (good.iter().flatten().cloned().collect(),
```

Comment:
Better to put in the values yourself rather than add the dependency (even just for development), I think.